index.d.ts

```diff
@@ -0,0 +1,6 @@
+/**
+Check if the character represented by a given [Unicode code point](https://en.wikipedia.org/wiki/Code_point) is [fullwidth](https://en.wikipedia.org/wiki/Halfwidth_and_fullwidth_forms).
+
+@param input - [Code point](https://en.wikipedia.org/wiki/Code_point) of a character.
+*/
```

Comment:
Can you include the example from the readme here when creating these definitions? Usually examples are the best documentation, and without, it's harder to use them.